Resources/Locale/en-US/chat/managers/chat-manager.ftl

```diff
@@ -52,28 +52,65 @@ chat-manager-admin-channel-name = ADMIN
 chat-speech-verb-suffix-exclamation = !
 chat-speech-verb-suffix-exclamation-strong = !!
 chat-speech-verb-suffix-question = ?
+chat-speech-verb-suffix-stutter = -
+chat-speech-verb-suffix-mumble = ..
 
 chat-speech-verb-default = says
 chat-speech-verb-exclamation = exclaims
 chat-speech-verb-exclamation-strong = yells
 chat-speech-verb-question = asks
+chat-speech-verb-stutter = stutters
+chat-speech-verb-mumble = mumbles
+
+chat-speech-verb-basic-1 = speaks
+chat-speech-verb-basic-2 = utters
 
 chat-speech-verb-insect-1 = chitters
-chat-speech-verb-insect-2 = flutters
+chat-speech-verb-insect-2 = chirps
+chat-speech-verb-insect-3 = clicks
+
+chat-speech-verb-winged-1 = flutters
+chat-speech-verb-winged-2 = flaps
+chat-speech-verb-winged-3 = buzzes
+
+chat-speech-verb-slime-1 = sloshes
+chat-speech-verb-slime-2 = burbles
+chat-speech-verb-slime-3 = oozes
 
-chat-speech-verb-slime = chirps
+chat-speech-verb-plant-1 = rustles
+chat-speech-verb-plant-2 = sways
+chat-speech-verb-plant-3 = creaks
 
-chat-speech-verb-robotic = states
-chat-speech-verb-reptilian = hisses
+chat-speech-verb-robotic-1 = states
+chat-speech-verb-robotic-2 = beeps
+chat-speech-verb-robotic-3 = reports
```

Comment:
remove `reports`, dont think it makes that much sense usually

Reply:
Sure, done.